docs/index.md

```diff
@@ -0,0 +1,145 @@
+
+* [Maven](https://maven.apache.org/)
+* [Gradle](https://gradle.org)
+
+**Quick hello world examples using SAM CLI**
+
+You can use [SAM](https://aws.amazon.com/serverless/sam/) to quickly setup a serverless project including AWS Lambda Powertools Java.
+
+```bash
+  sam init --location gh:aws-samples/cookiecutter-aws-sam-powertools-java
+```
+
+For more information about the project and available options refer to this [repository](https://github.com/aws-samples/cookiecutter-aws-sam-powertools-java/blob/main/README.md)
+
+=== "Maven"
+
+    ```xml
+    <dependencies>
+        ...
+        <dependency>
+            <groupId>software.amazon.lambda</groupId>
+            <artifactId>powertools-tracing</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.lambda</groupId>
+            <artifactId>powertools-logging</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        <dependency>
+            <groupId>software.amazon.lambda</groupId>
+            <artifactId>powertools-metrics</artifactId>
+            <version>1.2.0</version>
+        </dependency>
+        ...
+    </dependencies>
+    ```
+
+=== "Maven Configuration"
+
+    And configure the aspectj-maven-plugin to compile-time weave (CTW) the aws-lambda-powertools-java aspects into your project:
+
+    ```xml
+    <build>
+        <plugins>
+            ...
+            <plugin>
+                 <groupId>org.codehaus.mojo</groupId>
+                 <artifactId>aspectj-maven-plugin</artifactId>
+                 <version>1.11</version>
+                 <configuration>
+                     <source>1.8</source>
+                     <target>1.8</target>
+                     <complianceLevel>1.8</complianceLevel>
+                     <aspectLibraries>
+                         <aspectLibrary>
+                             <groupId>software.amazon.lambda</groupId>
+                             <artifactId>powertools-tracing</artifactId>
+                         </aspectLibrary>
+                         <aspectLibrary>
+                             <groupId>software.amazon.lambda</groupId>
+                             <artifactId>powertools-logging</artifactId>
+                         </aspectLibrary>
+                         <aspectLibrary>
+                             <groupId>software.amazon.lambda</groupId>
+                             <artifactId>powertools-metrics</artifactId>
+                         </aspectLibrary>
+                     </aspectLibraries>
+                 </configuration>
+                 <executions>
+                     <execution>
+                         <goals>
+                             <goal>compile</goal>
+                         </goals>
+                     </execution>
+                 </executions>
+            </plugin>
+            ...
+        </plugins>
+    </build>
```

Comment:
I think we need both parts of the pom to be present right? So I would put it into one file to reduce clicks and easier copy paste for the users.